libraryValidations/azuredeploy.json

```diff
@@ -0,0 +1,125 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "location": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().location]",
+      "metadata": {
+        "description": "Location for all resources."
+      }
+    },
+    "appConfigName": {
+      "type": "string",
+      "defaultValue": "[concat('appconfig', uniqueString(resourceGroup().id, deployment().name, utcNow()))]",
+      "metadata": {
+        "description": "Name of the App Configuration resource."
+      }
+    },
+    "appInsightsName": {
+      "type": "string",
+      "defaultValue": "[concat('appinsights', uniqueString(resourceGroup().id, deployment().name, utcNow()))]",
+      "metadata": {
+        "description": "Name of the Application Insights resource."
+      }
+    },
+    "workspaceName": {
+      "type": "string",
+      "defaultValue": "[concat('loganalytics', uniqueString(resourceGroup().id, deployment().name, utcNow()))]",
+      "metadata": {
+        "description": "Name of the Log Analytics workspace."
+      }
+    }
+  },
+  "resources": [
+    {
+      "type": "Microsoft.OperationalInsights/workspaces",
+      "apiVersion": "2023-09-01",
+      "name": "[parameters('workspaceName')]",
+      "location": "[parameters('location')]",
+      "properties": {}
+    },
+    {
+      "type": "Microsoft.Insights/components",
+      "apiVersion": "2020-02-02",
+      "name": "[parameters('appInsightsName')]",
+      "location": "[parameters('location')]",
+      "kind": "web",
+      "properties": {
+        "Application_Type": "web",
+        "WorkspaceResourceId": "[resourceId('Microsoft.OperationalInsights/workspaces', parameters('workspaceName'))]"
+      },
+      "dependsOn": [
+        "[resourceId('Microsoft.OperationalInsights/workspaces', parameters('workspaceName'))]"
+      ]
+    },
+    {
+      "type": "Microsoft.AppConfiguration/configurationStores",
+      "apiVersion": "2024-06-01-preview",
+      "name": "[parameters('appConfigName')]",
+      "location": "[parameters('location')]",
+      "sku": {
+        "name": "Standard"
+      },
+      "properties": {
+        "publicNetworkAccess": "Enabled",
+        "telemetry": {
+          "encryption": {},
+          "disableLocalAuth": false,
+          "softDeleteRetentionInDays": 7,
+          "enablePurgeProtection": false,
+          "dataPlaneProxy": {
+            "authenticationMode": "Local",
+            "privateLinkDelegation": "Disabled"
+          },
```

Comment:
shouldn't this be outside of the `telemetry` object?

Reply:
I don't think so, I copied this from the template in the azure portal.

Reply:
![image](https://github.com/user-attachments/assets/7a49ffe4-943e-4d11-a2d0-cecbcf2651d1)
does it look weird to you?

Reply:
encryption inside the telemetry object

Reply:
oh... didn't notice that. Bad copy/paste issue. fixed.